pkgs/tools/package-management/appimage-run/default.nix

```diff
@@ -10,7 +10,15 @@ in buildFHSUserEnv (fhsArgs // {
 
   runScript = writeScript "appimage-exec" ''
     #!${runtimeShell}
+    if [ $# -le 0 ]; then 
```

Comment:
The number of arguments passed couldn't be less than 0, so I would prefer `-eq` here.